.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
looks like you might have inadvertently added some personal IDE settings to the PR.

Reply:
I wish we all used vs code and agreed to committed settings... like auto format or not.... every w3c repo i work it, i have a bunch of settings disabling my default behavior, like format on save... but yea, probably a mistake  to commit this file.

Reply:
Didn't realize I had, I can delete.